src/main/java/com/redhat/red/build/finder/BuildFinder.java

```diff
@@ -461,6 +463,38 @@ public static void main(String[] args) {
                 throw new ParseException("Must specify at least one file");
             }
 
+            if (line.hasOption("debug")) {
+                ch.qos.logback.classic.Logger rootLogger = (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
+                rootLogger.setLevel(Level.DEBUG);
+
+                LoggerContext loggerContext = rootLogger.getLoggerContext();
+                loggerContext.reset();
+
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("com.redhat.red.build.koji")).setLevel(Level.WARN);
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.apache.commons.beanutils")).setLevel(Level.WARN);
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.apache.commons.vfs2")).setLevel(Level.WARN);
+                // XXX: Set to OFF due to <https://issues.apache.org/jira/browse/VFS-634>
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.apache.commons.vfs2.impl")).setLevel(Level.OFF);
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.apache.http")).setLevel(Level.WARN);
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.apache.kerby")).setLevel(Level.WARN);
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.commonjava.util.jhttpc")).setLevel(Level.WARN);
+                ((ch.qos.logback.classic.Logger) LoggerFactory.getLogger("org.commonjava.rwx")).setLevel(Level.WARN);
+
```

Comment:
I'm not very keen on duplicating this logger configuration here. Can we leave that in the logback.xml ?